flipperdevices/flipperzero-firmware
# [FL-2461] IR: change word from key to button

# What's new

- Renamed menu captions
- Renamed scene titles

# Verification 

- Open existing remote and click "Edit"
- Check that all nested captions haven't got word "key"

# Checklist (For Reviewer)
